packages/firebase_remote_config/lib/src/remote_config_settings.dart

```diff
@@ -0,0 +1,12 @@
+part of firebase_remote_config;
+
+/// RemoteConfigSettings can be used to configure how Remote Config operates.
+class RemoteConfigSettings {
+  /// When set to true developer mode is enabled, when set to false developer
+  /// mode is disabled. When developer mode is enabled fetch throttling is
+  /// relaxed to allow many more fetch calls per hour to the remote server than
+  /// the 5 per hour that is enforced when developer mode is disabled.
+  bool debugMode;
```

Comment:
This should be `final` (readonly) to match the native SDKs (e.g. [iOS](https://firebase.google.com/docs/reference/ios/firebaseremoteconfig/api/reference/Classes/FIRRemoteConfigSettings))

Reply:
Done